concent_api/core/tests/test_integration_accept_or_reject_results.py

```diff
@@ -15,11 +15,11 @@
 (CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY) = generate_ecc_key_pair()
 
 
-def _get_provider_account_status_true_mock(_):
+def _get_provider_account_status_true_mock(concent_rpc, requestor_eth_address, pending_value):  # pylint: disable=unused-argument
```

Comment:
I would suggest that in mock we just use `_` in the function signutares for names of unused values